Update GoW Theme for compatibility with phpBB3.

Deledrius opened the pull request:
This rewrites the existing theme to be compatible with phpBB3.  Due to extensive changes in how phpBB themes work, this is almost entirely a complete rewrite.

Deledrius committed:
Update theme to work with phpBB3.

Deledrius commented:
Here is a sample of the new theme's appearance:

![Screenshot 2021-10-15 at 05-19-38 Welcome to phpBB3 - Guild of Writers Beta](https://user-images.githubusercontent.com/187128/137498276-1790e514-3f5f-4d50-8bcc-3237479f2250.png)

Hoikas commented:
The contents of reported posts isn't being displayed on the moderator cp.
<img width="1351" height="353" alt="image" src="https://github.com/user-attachments/assets/0d3b7273-2a4a-4dba-bdcf-eed059902598" />

Hoikas commented:
The member list colors are questionable
<img width="710" height="207" alt="image" src="https://github.com/user-attachments/assets/a5d40a2a-ea95-4943-a48c-ec9b2f875636" />

Hoikas commented:
Icons for forums with subforums appear to be missing.

<img width="260" height="137" alt="image" src="https://github.com/user-attachments/assets/f7785a47-75d2-4563-9f4d-d05755981ec3" />

Hoikas commented:
PM colors are questionable

<img width="1298" height="466" alt="image" src="https://github.com/user-attachments/assets/1b455022-761b-48d7-9b19-f78780e5afb1" />

Deledrius commented:
Thanks!  Good idea to put those here.  I was keeping track in my head like a silly person.  I'll see about fixing those (and a few more).

Deledrius committed:
Fix typo in image filenames.

Deledrius committed:
Fix readability of notice text.

Deledrius committed:
Improve spacing between posts.

Deledrius committed:
Fix readability of attachment and info text.

Deledrius committed:
Hide default logo.

We have our logo displayed in the banner area above the forum and don't
need to repeat ourselves here.

Deledrius committed:
Increase brightness of GoW Blue for readability.

Deledrius committed:
Fix background color for PM display.

Deledrius committed:
Fix contrast on pagination and tables.

Deledrius committed:
Minor color tweaks.

Deledrius committed:
Add missing icons for Hot topics.

Deledrius committed:
Fix paginated navigation colors.

Deledrius committed:
Fix colors in Jumpbox.

Deledrius committed:
Normalize indentation.

Deledrius committed:
Fix button icon misalignment due to button font.

Deledrius committed:
Improve row hover and button readability.

Deledrius committed:
Use same color for all levels of nested quotes.

Hoikas commented:
The attachment view is a bit funky

<img width="2424" height="299" alt="image" src="https://github.com/user-attachments/assets/46066981-a52c-4576-b7f7-500d72077b39" />

Deledrius commented:
Yup, that'll need fixin'.  I'll get on that.

Deledrius committed:
Improve readability of attachment filelist.
